### PR TITLE
sci-visualization/fityk: Version bump to 1.3.1, EAPI bump 5 -> 6

### PR DESCRIPTION
**Pull request by g-braeunlich**

### CODE REVIEW

File: sci-visualization/fityk/fityk-1.3.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_4 )
```
Comment (SoapGentoo) [inline sci-visualization/fityk/fityk-1.3.1.ebuild]: can you please also test py3.5?

File: sci-visualization/fityk/fityk-1.3.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+IUSE="gnuplot nlopt readline python static-libs wxwidgets"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+CDEPEND="
+	>=sci-libs/xylib-1
+	>=dev-lang/lua-5.1:0
+	nlopt? ( sci-libs/nlopt )
+	python? ( ${PYTHON_DEPS} )
+	readline? ( sys-libs/readline:* )
```
Comment (SoapGentoo) [inline sci-visualization/fityk/fityk-1.3.1.ebuild]: if it links to readline, this needs to be `:0=`, it doesnt link, it needs to be `:0`

File: sci-visualization/fityk/fityk-1.3.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+	wxwidgets? ( x11-libs/wxGTK:3.0 )"
+DEPEND="${CDEPEND}
+	dev-libs/boost:=
+	dev-lang/swig"
+RDEPEND="${CDEPEND}
+	gnuplot? ( sci-visualization/gnuplot )"
+
+src_prepare() {
+	use python && python_copy_sources
+	default
```
Comment (SoapGentoo) [inline sci-visualization/fityk/fityk-1.3.1.ebuild]: swap around - currently, you can't patch stuff that affects python

File: sci-visualization/fityk/fityk-1.3.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+	nlopt? ( sci-libs/nlopt )
+	python? ( ${PYTHON_DEPS} )
+	readline? ( sys-libs/readline:* )
+	wxwidgets? ( x11-libs/wxGTK:3.0 )"
+DEPEND="${CDEPEND}
+	dev-libs/boost:=
+	dev-lang/swig"
+RDEPEND="${CDEPEND}
+	gnuplot? ( sci-visualization/gnuplot )"
+
```
Comment (SoapGentoo) [inline sci-visualization/fityk/fityk-1.3.1.ebuild]: `wxwidgets.eclass` generally requires you to call `setup-wxwidgets`:
```
pkg_setup() {
	use wxwidgets && setup-wxwidgets
}
```

File: sci-visualization/fityk/fityk-1.3.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+		}
+		python_foreach_impl python_compilation
+	fi
+}
+
+src_install() {
+	default
+	if use python; then
+		python_installation() {
+			emake DESTDIR="${ED%/}" -C "${BUILD_DIR}"/fityk install-pyexecLTLIBRARIES
```
Comment (SoapGentoo) [inline sci-visualization/fityk/fityk-1.3.1.ebuild]: you sure `DESTDIR="${ED%/}"` is correct? Autoconf/Automake usually honour --prefix and hence need to be used with `DESTDIR="${D}"` (or `DESTDIR="${D%/}"`), see https://wiki.gentoo.org/wiki/Project:Prefix/Technical_Documentation#The_variables_ED_and_EROOT

File: sci-visualization/fityk/fityk-1.3.1.ebuild [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+			rm "${ED%/}"/$(python_get_sitedir)/*.la || die
```
Comment (SoapGentoo) [inline sci-visualization/fityk/fityk-1.3.1.ebuild]: are there any `.la` files needed? I would just do the normal find `.la` stuff at the very end of `src_install` if all of them can be purged

Comment (g-braeunlich) [inline sci-visualization/fityk/fityk-1.3.1.ebuild]: I think in this case they are needed. There is no pc file and there is a libfityk.a and libfityk.la says that there are dependencies

File: sci-visualization/fityk/fityk-1.3.1.ebuild (hunk 2 above)
Comment (g-braeunlich) [inline sci-visualization/fityk/fityk-1.3.1.ebuild]: ldd reports that something links to readline -> :0=

File: sci-visualization/fityk/fityk-1.3.1.ebuild (hunk 1 above)
Comment (g-braeunlich) [inline sci-visualization/fityk/fityk-1.3.1.ebuild]: works

**Pull request closed without merging**